### PR TITLE
On download tab, we need to hide file download only if virtual study …

### PR DESCRIPTION
**Pull request by alisman**
…is true

# What? Why?
Fix # .

Changes proposed in this pull request:
- a
- b

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). If you are part of the
cBioPortal organization, notify the approprate team (remove inappropriate):

@cBioPortal/frontend
@cBioPortal/backend
@cBioPortal/devops

If you are not part of the cBioPortal organization look at who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them here:

### CODE REVIEW

File: portal/src/main/webapp/js/src/data_download.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ var DataDownloadTab = (function() {
             }
             
             // Hide download file links for multi-studies.
-            if (_processedProfiles.length > 1) {
+            if (window.QuerySession.isVirtualStudy) {
```
Comment (alisman) [inline portal/src/main/webapp/js/src/data_download.js]: @jjgao please confirm the logic here.  Virtual study will be true if study is virtual cohort OR if multi-study.

**Comment by kalletlak** [comment]
@alisman @jjgao I remember that we had this feature working even for virtual study. Let me try commenting that code and see if it works

**Comment by alisman** [comment]
@kalletlak I think I fixed that.  You'll see latest version of this PR only looks at number of studies.  Is that what you mean?

File: portal/src/main/webapp/js/src/data_download.js [hunk 2]
```diff
@@ -212,16 +212,16 @@ var DataDownloadTab = (function() {
...
+            //     _processedProfiles = [];
+            //     $.each(_.groupBy(profiles, function(_profile){ return _profile.genetic_alteration_type}), function(k,v){
+            //         var _p = {name:_profileNames[k], profileIds:v.map(function(d) { return d.id; }).join(','), alterationType:k,fileName:k.toLowerCase()}
+            //         _processedProfiles.push(_p);
+            //     });
+            // }
             
             // Hide download file links for multi-studies.
-            if (_processedProfiles.length > 1) {
+            if (QuerySession.getCancerStudyIds().length > 1) {
```
Comment (adamabeshouse) [inline portal/src/main/webapp/js/src/data_download.js]: lets use the new store for this?

**Pull request merged**